etesync_dav/radicale_main/__init__.py

```diff
@@ -71,17 +74,28 @@ def run(passed_args=None):
                 del kwargs["internal"]
 
             if kwargs["type"] == bool:
-                del kwargs["type"]
-                kwargs["action"] = "store_const"
-                kwargs["const"] = "True"
-                opposite_args = kwargs.pop("opposite", [])
-                opposite_args.append("--no%s" % long_name[1:])
-                group.add_argument(*args, **kwargs)
-
-                kwargs["const"] = "False"
-                kwargs["help"] = "do not %s (opposite of %s)" % (
-                    kwargs["help"], long_name)
-                group.add_argument(*opposite_args, **kwargs)
+                if int(version_minor) >= 1:  # Changed since 3.1.0
```

Comment:
```suggestion
                if int(version_major) >= 3 and int(version_minor) >= 1:  # Changed since 3.1.0
```
I know we are version 3 only anyway, but still worth being explicit.